devimint/src/main.rs

```diff
@@ -381,31 +381,31 @@ async fn cli_tests(dev_fed: DevFed) -> Result<()> {
         .as_str()
         .map(|s| s.to_owned())
         .unwrap();
-    let client_ng_reissue_amt = cmd!(fed, "reissue", reissue_notes)
+    let client_reissue_amt = cmd!(fed, "reissue", reissue_notes)
```

Comment:
Did we delete the legacy client code in this test? I remember explicitly naming these with `ng` because it was really easy to use the same name as the legacy client, which resulted in weird issues (either errors or silent successes)

Reply:
The legacy CLI tests have been gone for a while. The only legacy client tests left are pure rust tests.